node/main-extract.ts

```diff
@@ -540,6 +501,7 @@ function spawn_ffmpeg_and_run(
 
   return new Promise((resolve, reject) => {
 
+    // Uncomment things in this method to check how long extraction takes
```

Comment:
~Uncomment things in this method~ Pass `--debug` when starting to check how long extraction takes

Reply:
This commenting is very specific -- only to see how long extraction takes place. It is not needed during regular debug, and I'd rather not spam-up the console logs with every debug statement we have unless one wants to see them. 🤔